applications/nrf_desktop/doc/ble_adv.rst

```diff
@@ -3,4 +3,56 @@
 BLE advertising module
 ######################
 
-Use the ``ble_adv`` module to control the BLE advertising.
+Use the BLE advertising module to control the Bluetooth LE advertising.
+
+Module Events
+*************
+
++------------------+------------------------------+-------------+---------------------------+-------------------+
+| Source Module    | Input Event                  | This Module | Output Event              | Sink Module       |
++==================+==============================+=============+===========================+===================+
+| :ref:`ble_state` | ``ble_peer_event``           | ``ble_adv`` |                           |                   |
++------------------+------------------------------+             |                           |                   |
+| :ref:`ble_bond`  | ``ble_peer_operation_event`` |             |                           |                   |
++------------------+------------------------------+             +---------------------------+-------------------+
+|                  |                              |             | ``ble_peer_search_event`` | :ref:`led_state`  |
++------------------+------------------------------+-------------+---------------------------+-------------------+
+
+
+Configuration
+*************
+
+The module requires the basic Bluetooth configuration, as described in the Bluetooth guide.
+
+Set the ``CONFIG_DESKTOP_BLE_ADVERTISING_ENABLE`` to enable the BLE advertising module for the nRF Desktop peripheral device.
+
+You must define the short device name used for advertising (``CONFIG_DESKTOP_BLE_SHORT_NAME``), the Bluetooth device name (:option:`CONFIG_BT_DEVICE_NAME`) and the Bluetooth device appearance (:option:`CONFIG_BT_DEVICE_APPEARANCE`).
+
+By default, the module uses indirect advertising.
+Set the ``CONFIG_DESKTOP_BLE_DIRECT_ADV`` option to use direct advertising.
+
+.. tip::
+   The nRF Desktop peripheral will not advertise directly towards central that uses Resolvable Private Address.
+   The peripheral does not check if the remote device supports address resolution of directed advertisements - it does not read the Central Address Resolution GATT characteristic of the central.
```

Comment:
I am not sure if because is a good word here.
You check if the remote device supports the address resolution of directed advertisements by reading the Central Address Resolution GATT characteristic of the central

Reply:
I will update this.